Email field  validation on contact form

lmunozm1702 opened the pull request:
## Implement the validation for the email field in lowercase.

The message must appear up to the field with a red background, as follow:

![image](https://user-images.githubusercontent.com/53025062/195712339-5731699a-0192-4d35-80e5-b8aac1fdf694.png)

### When the email is correct, the message is removed (if it already exists), and submit the form.

lmunozm1702 committed:
Create listener

HevarKas committed:
driver:hevar navigator:luis doing the validation for function validateemail

lmunozm1702 committed:
Email javascript validation

- Add index_contact.js
- Update style.css

To implement the Javascript client-side validation for email, this field must be in uppercase.

lmunozm1702 committed:
Solve linter errors

lmunozm1702 committed:
Solve linter issues

KDlamini commented on assets/js/index_contact.js:
```diff
@@ -0,0 +1,35 @@
+  if (errorDiv) {
+    errorDiv.remove();
+  }
+
+  if (!validateEmail(form.elements['contact-email'])) {
+    setMessage('contact-email', 'Ooops!, please write your email in lowercase!', 'error');
+  } else {
+    form.submit();
```
Please make sure that the error message is near the submit button as per project requirements.

![image](https://user-images.githubusercontent.com/38088818/195715430-eb41223b-814c-45d0-a552-3216da03b94d.png)

lmunozm1702 replied:
It's done:

![image](https://user-images.githubusercontent.com/53025062/195721806-1ca30eb2-d032-4618-9558-8cb82905ec80.png)

KDlamini reviewed (changes requested):
### Hi @lmunozm1702 🙋‍♀️,  

Great work so far! 💯 
There are some issues that you still need to work on improving your project according to the requirements. You are almost there! 💪🏾 

## Highlights 🌞 

✔️  Email validation works as expected
✔️  No linter errors

## Required Changes ♻️
 
**Kindly look through the comments below 👇 for required changes**

Cheers 🥂  and Happy coding!!! 👯 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me @KDlamini  in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

lmunozm1702 committed:
Solve code review issue - Error message ubication.

Update style.css, index_contact.js and index.html to move the error message near the button.

lmunozm1702 committed:
Fix linter error.

jerryowusu reviewed (approved):
Hi Team :wave: 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

![wow approved](https://user-images.githubusercontent.com/65233860/195721854-6db88cb6-e7e6-495c-bc63-374fe1c00d7d.gif)


## Highlights
:heavy_check_mark: All linters are passing
:heavy_check_mark: Followed GitHub flow
:heavy_check_mark: Good commit messages

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
